accounts/models.py

```diff
@@ -1,3 +1,32 @@
 from django.db import models
 
-# Create your models here.
+from django.contrib.auth.models import User
+
+
+class Profile(models.Model):
+    SEXO_CHOICES = (
+        (u'masculino', 'Masculino'),
+        (u'feminino', 'Femninino'),
+    )
+
+    nome = models.CharField(max_length=100)
+    sexo = models.CharField(max_length=10, choices=SEXO_CHOICES)
+    email = models.EmailField(max_length=50)
+    telefone = models.BigIntegerField()
```

Comment:
@AyronFelipe como postado pelo @rogerio410 o campo para telefone poderia ter algum tipo de validação , o BigIntegerField seria para números grandes,  o telefone é algo mais estruturado.

E para as validação há o uso das expressões regulares, que casam um padrão de estrutura , como o caso do telefone. 

Se for para gerar um campo de formulário depois pode usar o ```RegexField``` , mas por enquanto dá de usar o ```RegexValidator``` para casar a estrutura de um telefone , recebendo a regex e uma mensagem de erro nos parâmetros.

Para o formato simples : 9999-9999 poderia usar um : ^\d{4}-\d{4}$ para que a string comece com um número em ocorrência de 4 caracteres seguidos de um "-" e depois novamente 4 ocorrências de números.

Aqui tem uns exemplos de validações para formatos de telefones :

[Formatos de telefones](https://github.com/Marlysson/Programacao-Corporativa/tree/master/Uso%20de%20Regex#validação-5)